e2e/next-sandbox/turbo.json

```diff
@@ -0,0 +1,9 @@
+{
+  "$schema": "https://turborepo.org/schema.json",
+  "extends": ["//"],
+  "pipeline": {
+    "test": {
+      "cache": false
+    }
+  }
+}
```

Comment:
The point of the E2E test is that it's testing the backend. If the backend changes, but the front-end does not, Turbo will think nothing changed, and serve the cache. This makes no sense for the E2E tests like it does for normal unit tests.